base/exports.jl

```diff
@@ -70,6 +70,9 @@ export
     OrdinalRange,
     Pair,
     PartialQuickSort,
+    OncePerProcess,
+    OncePerTask,
+    OncePerThread,
```

Comment:
This should probably not be exported?

Reply:
*This = OncePerThread only. And still public, just not exported.

Reply:
It's based on the idea that OncePerThread should be used very rarely (perhaps never in the absence of FFI)

Reply:
Though, yes, it is fairly arbitrary, and I see the appeal to putting all three at the same level.